app/views/session_timeout/_ping.js.erb

```diff
@@ -1,6 +1,7 @@
 var frequency = <%= frequency %> * 1000;
 var warning = <%= warning %> * 1000;
 var start = <%= start %> * 1000;
+var timeoutPath = "<%= timeout_path %>";
```

Comment:
It _should_ be fine as-is, but this sort of variable output makes me uneasy (existed previously as well).

Maybe:

```suggestion
var timeoutPath = <%= timeout_path.to_json %>;
```

Related, from our new ERBLint: https://github.com/Shopify/erb-lint#ErbSafety